### ISSUE TITLE
Add a option to disable the automatic prepend of field label?

### ISSUE THREAD

**Issue by ashuradev** [vlucas/valitron#311]
It would be nice to have an option to disable automatic field label prepend.

Example:

```php
$v->setLabelPrepend(false);

// language.php
return [
    'required' => 'Este campo é obrigatório'
];
```

With this option, the Validator does not add `{field}` to the message, so the outgoing message is:
`Este campo é obrigatório`

And not this:

`{field} Este campo é obrigatório`

Thanks guys :-)

**Comment by TomOudeRengerink** [comment]
@ashuradev I have opened a PR for this feature in #325

**Comment by ashuradev** [comment]
Closed by #325. Thanks!!
